testing/tests/test_var_fps.py

```diff
@@ -69,15 +62,22 @@ def test_classification(self):
                 rend_path = rendition_dir + os.path.sep + filename
                 if not os.path.exists(rend_path):
                     continue
-                res = self.verify_file(src, rend_path)
+                np.random.seed(123)
+                random.seed(123)
+                start = timeit.default_timer()
+                verification_result = verifier.verify(src, [{'uri': rend_path}])
+                tamper = float(verification_result[0]["tamper"])
```

Comment:
I think this will overwrite the `tamper` variable assigned on L60 which indicates if the rendition is actually tampered while `verification_result[0]["tamper"]` indicates if the meta model classified the rendition as tampered.

Reply:
Good catch, fixed.